src/filters/FiltersActions.js

```diff
@@ -29,11 +32,13 @@ export const receiveSections = (data) => {
 };
 
 export const fetchSections = () => {
-  return (dispatch) => {
+  return (dispatch, getState) => {
     dispatch(requestSections());
 
+    const app = getState().app;
+
     /* xenia_package */
-    fetch(window.xeniaHost + '/1.0/exec/dimension_section_list', authXenia())
+    fetch(app.xeniaHost + '/1.0/exec/dimension_section_list', authXenia())
```

Comment:
Template interpolation looks good for here

` fetch(`${app.xeniaHost}/1.0/exec/dimension_section_list`, authXenia())`